### ISSUE TITLE
Simulating multiple genetically correlated phenotypes

### ISSUE THREAD

**Issue by fmorgante** [privefl/bigsnpr#419]
Hi Florian, thank you for a great package! I think it would be very useful if `snp_simuPheno` could be extended to simulate multiple genetically correlated phenotypes. I guess the framework could be made as flexible as one wants, but a great first step would be to allow for the effects to be drawn from a multivariate normal with mean 0 and a user-provided covariance matrix across traits, and independent residuals across traits.

Do you think this is something you are interested in implementing?

**Comment by fmorgante** [comment]
Yes, absolutely. I guess you could start by imposing the same causal variants across traits, but allowing an arbitrary number of traits. The user would then provide the desired genetic correlation matrix

**Comment by fmorgante** [comment]
That sounds great!

**Comment by privefl** [comment]
Do you want to try to implement it?

**Comment by fmorgante** [comment]
I do not have bandwidth at the moment, but I may be able to get to it later in the summer (if you or someone else does not do it before). I suggest we keep this issue open until someone implements this. What do you think?

**Comment by privefl** [comment]
Same. Sounds a good idea.
